content/docs/reference/types/rich-text.md

```diff
@@ -20,3 +20,6 @@ type RichTextField = {
 
 To add an embedded template quickly enter `/`, this will present you with the embedable objects,
 filtering them out as you type.
+
+#### Default Values
+> Currently, if setting a default value for a rich-text field, you must provide the document AST. 
```

Comment:
@scottgallant  Would it be better to say it is not supported? 

the AST syntax is not super initiative and might cause more confusion.